chrome-launcher.ts

```diff
@@ -128,9 +128,12 @@ export class Launcher {
       `--user-data-dir=${isWsl ? toWinDirFormat(this.userDataDir) : this.userDataDir}`
     ]);
 
-    if (this.enableExtensions) {
-      flags = flags.filter(flag => flag !== '--disable-extensions');
-    }
+    flags = flags.filter(flag => {
+      if (this.defaultIgnoreFlags) {
```

Comment:
TypeScript complains if this if statement is missing.
Due to strict null type checking